libraries/botbuilder/src/streamingHttpClient.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * @module botbuilder
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IStreamingTransportServer, StreamingRequest } from 'botframework-streaming';
+import { WebResource, HttpOperationResponse, HttpClient } from 'botframework-connector/node_modules/@azure/ms-rest-js';
```

Comment:
The @azure/ms-rest-js dependency in the package.json of botbuilder is the same as in botframework-connector. Please have this import just pull from @azure/ms-rest-js.